rules.d/99-systemd.rules.in

```diff
@@ -18,10 +18,6 @@ SUBSYSTEM=="ubi", TAG+="systemd"
 
 SUBSYSTEM=="block", TAG+="systemd"
 
-# We can't make any conclusions about suspended DM devices so let's just import previous SYSTEMD_READY state and skip other rules
-SUBSYSTEM=="block", ENV{DM_SUSPENDED}=="1", IMPORT{db}="SYSTEMD_READY"
-SUBSYSTEM=="block", ENV{DM_SUSPENDED}=="1", GOTO="systemd_end"
```

Comment:
this was added in 466266c1728e3b53b7020e4c0165c3e88576b805, @msekletar hopefully knows why.

Reply:
I know. I've studied the history. My patch set keeps this behavior, it just replaces `DM_SUSPENDED` with `DM_UDEV_DISABLE_OTHER_RULES_FLAG`.

Wrt the `GOTO`, that's really a minor thing. As I wrote in the commit message, I think it isn't necessary because the rest of the rules doesn't apply to the devices in question anyway.

Reply:
@msekletar, am I assuming correctly that the fix for that bug was the `IMPORT{db}="SYSTEMD_READY"` for a suspended device? 

My patch keeps this logic, it just replaces the condition on `DM_SUSPENDED` by a `DM_UDEV_DISABLE_OTHER_RULES_FLAG`. `DM_SUSPENDED` implies `DM_UDEV_DISABLE_OTHER_RULES_FLAG`. Would you concur that my change wouldn't re-introduce breakage for that bug?

Reply:
To verify, I just tested the reproducer from https://bugzilla.redhat.com/show_bug.cgi?id=1985288 with my patches from this PR applied. The issue from the bug did not occur.